apps/portal/components/containers/dashboardLayout/elements/SidebarMenuItems/SidebarMenuItem.tsx

```
@@ -9,32 +18,138 @@ interface ISidebarMenuItemProps {
   className?: string;
   onClick?: () => void;
 }
+export function Chevron(props: any) {
+  return (
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
+      width="1em"
+      height="1em"
+      viewBox="0 0 1024 1024"
+      {...props}
+    >
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)